charts/partials/templates/_proxy.tpl

```diff
@@ -16,6 +16,18 @@ env:
   value: {{ternary "localhost.:8086" (printf "linkerd-dst-headless.%s.svc.%s.:8086" .Values.namespace .Values.clusterDomain) (eq (toString .Values.proxy.component) "linkerd-destination")}}
 - name: LINKERD2_PROXY_DESTINATION_PROFILE_NETWORKS
   value: {{.Values.clusterNetworks | quote}}
+- name: _pod_name
+  valueFrom:
+    fieldRef:
+      fieldPath: metadata.name
+- name: _pod_ns
+  valueFrom:
+    fieldRef:
+      fieldPath: metadata.namespace
+- name: _pod_nodeName
+  valueFrom:
+    fieldRef:
+      fieldPath: spec.nodeName
```

Comment:
perhaps just put _all_ of the `_vars` at the top of the env so there can be no confusion?

Reply:
@olix0r Some of the other `_vars` are gated based on identity and which makes it a bit confusing to have them default. I can clean this up as part of #6702